assignment-5/updateUser.js

```diff
@@ -0,0 +1,14 @@
+module.exports.updateUser = (app, user) => {
+  app.put('/updateUser', (req, res) => {
+    var searchID = require('./findingID');
+    var receivedID = req.body.id;
+    var userID = searchID.findingID(user, receivedID);
+    if (userID != -1) {
```

Comment:
@JainVayam If the 'IF statement is one line' then you can write it without using the curly braces. like below,
 if (userID != -1) user[userID] = req.body;